ports/libsimpleservo/jniapi/src/lib.rs

```diff
@@ -560,6 +560,31 @@ impl HostTrait for HostCallbacks {
         )
         .unwrap();
     }
+
+    fn on_media_session_set_position_state(
+        &self,
+        duration: Option<f64>,
+        position: Option<f64>,
+        playback_rate: Option<f64>,
+    ) {
+        info!(
+            "on_media_session_playback_state_change ({:?}, {:?}, {:?})",
+            duration, position, playback_rate
+        );
+        let env = self.get_env().unwrap();
+        let duration = duration.map(|v| JValue::Float(v as jfloat)).unwrap_or(null);
+        let position = position.map(|v| JValue::Float(v as jfloat)).unwrap_or(null);
+        let playback_rate = playback_rate
+            .map(|v| JValue::Float(v as jfloat))
+            .unwrap_or(null);
```

Comment:
These `unwrap_or(null)`s won't build as it is expecting a `JSValue`. Anyway, as I pointed out before, I think we should not use `Option`s here and always pass the updated value for all the `MediaPositionState` members.

Reply:
fixed it.